storage/innobase/xtrabackup/test/inc/common.sh

```diff
@@ -2,7 +2,11 @@ set -eu
 
 function xtrabackup()
 {
+  if [ -z ${WITH_RR+x} ]; then
```

Comment:
Is this new option we are introducing? we should mention about it in run.sh --help